reactor-test/src/main/java/reactor/test/ValueFormatters.java

```diff
@@ -48,6 +49,13 @@
 
 	private ValueFormatters() {}
 
+	/**
+	 * Default {@link Duration#toString() Duration} {@link ToStringConverter} that removes the PT prefix and
+	 * switches {@link String#toLowerCase() to lower case}.
+	 */
+	public static final ToStringConverter DURATION_CONVERTER = new ClassBasedToStringConverter<Duration>(
```

Comment:
WDYT about putting a new line after each lambda? `true, d` was a bit hard to read, confused with `(a, b) ->`-like lambda declaration :D